src/notificationcenter.cs

```diff
@@ -47,6 +64,107 @@ interface UIVibrancyEffect_NotificationCenter {
 		[Static, Export ("notificationCenterVibrancyEffect")]
 		UIVibrancyEffect NotificationCenterVibrancyEffect ();
 	}
+#endif
+
+#if MONOMAC
+	[Mac (10,10)]
+	[BaseType (typeof(NSViewController), Delegates=new string [] { "Delegate" }, Events=new Type [] { typeof (NCWidgetListViewDelegate)})]
+	interface NCWidgetListViewController
+	{
+		[Wrap ("WeakDelegate")]
+		[Protocolize]
+		NCWidgetListViewDelegate Delegate { get; set; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; set; }
+
+		[Export ("contents", ArgumentSemantic.Copy)]
+		NSViewController[] Contents { get; set; }
 
+		[Export ("minimumVisibleRowCount", ArgumentSemantic.Assign)]
+		nuint MinimumVisibleRowCount { get; set; }
 
+		[Export ("hasDividerLines")]
+		bool HasDividerLines { get; set; }
+
+		[Export ("editing")]
+		bool Editing { get; set; }
+
+		[Export ("showsAddButtonWhenEditing")]
+		bool ShowsAddButtonWhenEditing { get; set; }
+
+		[Export ("viewControllerAtRow:makeIfNecessary:")]
+		NSViewController GetViewController (nuint row, bool makeIfNecesary);
+
+		[Export ("rowForViewController:")]
+		nuint GetRow (NSViewController viewController);
+	}
+
+	[Mac (10, 10)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface NCWidgetListViewDelegate
+	{
+		[Abstract]
+	[Export ("widgetList:viewControllerForRow:"), DelegateName ("NCWidgetListViewGetController"), DefaultValue (null)]
+		NSViewController GetViewControllerForRow (NCWidgetListViewController list, nuint row);
+
+	[Export ("widgetListPerformAddAction:"), DelegateName ("NCWidgetListViewController")]
+		void PerformAddAction (NCWidgetListViewController list);
+
+	[Export ("widgetList:shouldReorderRow:"), DelegateName ("NCWidgetListViewControllerShouldReorderRow"), DefaultValue (false)]
+		bool ShouldReorderRow (NCWidgetListViewController list, nuint row);
+
+	[Export ("widgetList:didReorderRow:toRow:"), EventArgs ("NCWidgetListViewControllerDidReorder"), DefaultValue (false)]
+		void DidReorderRow (NCWidgetListViewController list, nuint row, nuint newIndex);
+
+	[Export ("widgetList:shouldRemoveRow:"), DelegateName ("NCWidgetListViewControllerShouldRemoveRow"), DefaultValue (false)]
+		bool ShouldRemoveRow (NCWidgetListViewController list, nuint row);
+
+	[Export ("widgetList:didRemoveRow:"), EventArgs ("NCWidgetListViewControllerDidRemoveRow"), DefaultValue (false)]
+		void DidRemoveRow (NCWidgetListViewController list, nuint row);
+	}
+
+	[Mac (10,10)]
+	[BaseType (typeof(NSViewController), Delegates=new string [] { "Delegate" }, Events=new Type [] { typeof (NCWidgetSearchViewDelegate)})]
+	interface NCWidgetSearchViewController
+	{
+		[Wrap ("WeakDelegate")]
+		[Protocolize]
```

Comment:
Don't use `[Protocolize]` in new code, just bind using the protocol (`INCWidgetSearchViewDelegate`) instead.